app/views/devise/two_factor_authentication/show.html.slim

```diff
@@ -7,17 +7,17 @@ p.mt-tiny.mb0#2fa-description
 = simple_form_for(@otp_delivery_selection_form,
     url: otp_send_path,
     method: :get,
-    role: 'form', class: 'mt3 sm-mt4') do |f|
+    role: 'form', html: { class: 'mt2 sm-mt3' }) do |f|
   - if reauthn?
     = f.input :reauthn, as: :hidden, input_html: { value: 'true' }
   .mb3
-    fieldset.border-none
+    fieldset.border-none.mxn1
```

Comment:
I think these look fine without the indent removed, especially on mobile, but thats purely personal preference.